core/src/main/java/io/confluent/kafka/schemaregistry/leaderelector/kafka/KafkaGroupLeaderElector.java

```diff
@@ -97,8 +99,11 @@ public KafkaGroupLeaderElector(SchemaRegistryConfig config,
           CommonClientConfigs.METRIC_REPORTER_CLASSES_CONFIG,
           MetricsReporter.class
       );
-      reporters.add(new JmxReporter(JMX_PREFIX));
-
+      reporters.add(new JmxReporter());
+      for (MetricsReporter reporter : reporters) {
```

Comment:
If we use this Metrics constructor:
https://github.com/confluentinc/ce-kafka/blob/d6ecf6a2d2d34c7fbea13880dea51585f87c7122/clients/src/main/java/org/apache/kafka/common/metrics/Metrics.java#L132
Then there is no need to do the iteration here.

Reply:
Fixed, thank you !